scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamic.scala

```diff
@@ -117,7 +117,7 @@ final case class ScalafmtDynamic(
       configsCache
         .getOrAddToCache(
           configPath,
-          _.right.exists(_._2.compareTo(currentTimestamp) != 0)
+          _.fold(_ => true, _._2.compareTo(currentTimestamp) != 0)
```

Comment:
A unfortunate scala 2.11 compatible alias of the more readable:
```suggestion
          _.forall(_._2.compareTo(currentTimestamp) != 0)
```